src/appleseed/renderer/kernel/lighting/backwardlightsampler.cpp

```diff
@@ -80,6 +80,11 @@ BackwardLightSampler::BackwardLightSampler(const Scene& scene, const ParamArray&
     // Prepare the CDFs for sampling.
     if (m_non_physical_lights_cdf.valid())
         m_non_physical_lights_cdf.prepare();
+    
+    // Build the light-tree.
```

Comment:
Let's spell it *light tree*, and keep the hyphen when talking about "light-tree lights".